chrome/content/coniks/context.js

```diff
@@ -6,56 +6,46 @@ Cu.import("chrome://otr/content/helpers.js");
 
 var OS;
 
-function ConiksContext(opts) {
-  if (!(this instanceof ConiksContext)) {
-    return new ConiksContext(opts);
-  }
-  opts = opts || {};
+class ConiksContext {
+  constructor(opts) {
+    opts = opts || {};
 
-  if (opts.account === null) {
-    return null;
-  }
-  this.account = opts.account;
-  if (opts.fingerprint !== undefined || opts.fingerprint !== null) {
-    this.fingerprint = opts.fingerprint;
+    if (opts.account === null) {
+      return null;
+    }
+    this.account = opts.account;
+    if (opts.fingerprint !== undefined || opts.fingerprint !== null) {
+      this.fingerprint = opts.fingerprint;
```

Comment:
No, it is the fingerprint of the account. It will be used to send the registration request (see: https://github.com/c633/ctypes-otr/blob/coniks-registration/chrome/content/coniks/coniks.js#L125-L131). It's also will be used to verify the returned TB (or monitoring lookup), as in https://github.com/c633/ctypes-otr/blob/coniks-registration/chrome/content/coniks/context.js#L51-L55. Hopefully it makes sense to you.